testground/benchmark/benchmark/stateless.py

```diff
@@ -192,21 +210,29 @@ def run(outdir: str, datadir: str, cronosd, global_seq):
     home = datadir / group / str(group_seq)
 
     try:
-        return do_run(home, cronosd, group, global_seq, cfg)
+        return do_run(home, cronosd, group, cfg, num_workers, sleep)
     finally:
-        # collect outputs
-        output = Path("/data.tar.bz2")
-        with tarfile.open(output, "x:bz2") as tar:
-            tar.add(home, arcname="data", filter=output_filter(group, group_seq))
-        outdir = Path(outdir)
-        if outdir.exists():
-            assert outdir.is_dir()
-            filename = outdir / f"{group}_{group_seq}.tar.bz2"
-            filename.unlink(missing_ok=True)
-            shutil.copy(output, filename)
-
-
-def do_run(home: str, cronosd: str, group: str, global_seq: int, cfg: dict):
+        if group == VALIDATOR_GROUP:
```

Comment:
why this condition, is the output too big?

Reply:
just to kill fullnodes as soon as possible, I think there's competing in cluster as well